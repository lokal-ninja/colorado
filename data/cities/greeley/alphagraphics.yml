id: 9738223270
type: node
latitude: 40.4219
longitude: -104.76055
postcode: "80634"
city: Greeley
street: "West 10th Street"
housenumber: 4895
phone: "+9703560606"
opening_hours: ""
website: "http://alphagraphics.com"