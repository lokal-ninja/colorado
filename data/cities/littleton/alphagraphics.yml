id: 9310010110
type: node
latitude: 39.59132
longitude: -104.99037
postcode: ""
city: Littleton
street: "South Broadway"
housenumber: 6905
phone: ""
opening_hours: ""
website: ""
