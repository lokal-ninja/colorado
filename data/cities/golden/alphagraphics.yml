id: 9162363216
type: node
latitude: 39.75475
longitude: -105.22148
postcode: "80401"
city: Golden
street: "Miner's Alley"
housenumber: 1217
phone: ""
opening_hours: ""
website: ""
